src/app/domain/EloInfo.ts

```diff
@@ -0,0 +1,12 @@
+import { FirestoreJSONObject } from '../utils/utils';
+
+/** Player Elo Info
+  * All the info about the elo that end up in a sub-collection inside user's document
```

Comment:
on capitalise pas Elo ?
+ inside the user's document

Reply:
C'pas un TLA, c'est un prénom utilisé dans un contexte pas de prénom. Elo et elo sont à débattre mais pas ELO :D